assignments/week_2/project_2/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>  // Gives us access to std::cin and std::cout
+#include <string>  // Gives us access to std::string and std::getline()
+
+// TODO Define your light controller state machine class here
+
+int main()
+{
+    // TODO Initialize your state machine class here
+    while(true)
+    {
+        std::string input;
+        std::getline(std::cin, input);
+        if(input.size() != 2)
+            break;
+
+        // TODO Parse the input digits
```

Comment:
Are we having them do the parsing themselves? I'm not opposed to that, but I think it would be good to provide them with a little bit more structure here.

Reply:
Maybe we could give them a `struct` declaration that contains the six booleans?